components/contact/ContactInputs.tsx

```diff
@@ -0,0 +1,114 @@
+"user client";
+
+import toast from "react-hot-toast";
+import emailjs from "@emailjs/browser";
+import { useRef, MutableRefObject, useState } from "react";
+
+import InputField from "@/components/contact/InputField";
+
+const ContactInputs = () => {
+  const form: MutableRefObject<HTMLFormElement | null> = useRef(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  // NOTE - Cannot make it exportable because of the use of IsLoading
+  const inputField = [
```

Comment:
It seems that you have multiple fields so a better name for this constant would be `inputFields`.